app/views/application_drafts/_student_fields.html.slim

```diff
@@ -55,7 +53,7 @@
 
   = s.input :application_learning_history, as: :character_limited, label: 'Please summarize what you have been doing to learn programming', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_learning_history) }, hint: (student_field_error_hint(index, application_draft, :application_learning_history) || 'E.g., entered a study group, learned Rails by doing exercises, etc.')
 
-  = s.input :application_code_samples, as: :text, label: 'Please provide examples of your code, preferably in the language of the project you\'re applying for', input_html: { rows: 4, disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_code_samples) }, hint: (student_field_error_hint(index, application_draft, :application_code_samples) || 'E.g., links to completed coding challenge provided by the mentor (if present), commits or pull requests on Github or other concise examples of your code. Links to GitHub profiles and repositories won\'t be considered.')
+  = s.input :application_code_samples, as: :character_limited, label: "Please provide links to examples as well as short explanations of your code, preferably in the language of the project you're applying for", input_html: { rows: 4, disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_code_samples) }, hint: (student_field_error_hint(index, application_draft, :application_code_samples) || "E.g., links to completed coding challenge provided by the mentor (if present), commits or pull requests on Github or other concise examples of your code. Links to GitHub profiles and repositories won't be considered. Please do not directly copy-paste code here.")
```

Comment:
Same here: When rebasing this to `master` please be aware that I extracted all the labels into translation files.